datacollectiondemo/source/c/CMakeLists.txt

```diff
@@ -15,23 +15,31 @@
 #
 
 cmake_minimum_required(VERSION 2.8.8)
-
-project (C-SDK-sample)
-enable_language(C)
+project (C-SDK-sample LANGUAGES C)
 
 find_package (OpenSSL REQUIRED)
 
+set(APP_NAME "demo_client")
+set(KAA_SDK_DIR "client-c")
+
+execute_process(COMMAND ${CMAKE_COMMAND} -E make_directory ${KAA_SDK_DIR}
+                WORKING_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/libs/kaa"
```

Comment:
Because It's convenient having tar.gz separately from unpacked sdk. It could be easily removed